docs/source/_templates/home.html

```diff
@@ -82,19 +83,19 @@
             <div id="navbar-menu" class="navbar-menu is-static">
 
                 <div class="navbar-end">
-                    <a href="installation.html" id="navbar-item" class="navbar-item is-secondary">
```

Comment:
Good coding practice,
removed duplicate id attribute as that should be unique across elements